src/impi/core.clj

```diff
@@ -0,0 +1,12 @@
+(ns impi.core)
+
+(defmacro add-props! [& {:as coll}]
```

Comment:
The idea with the macro was to turn it into a symbolic lookup instead of using aset. From my understanding aset messes with the clojure compilers ability to rename keys. And since that function touches all Pixi objects that could lead to a large size increase.